src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 19px 0 22px;
+  }
+
+  &__item {
+    flex-direction: row;
+    align-self: center;
+    margin-top: 0;
+
+    &:not(:last-child) {
+      padding-right: 20px;
+    }
+  }
+
+  &__link {
+    font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
You've already set it in `body` - delete it here (or set `inherit` if it doesn't work properly)